### PR TITLE
prow.k8s.io bumping PR automatically merged

### PR DESCRIPTION
**Pull request by chaodaiG**
This PR enables PR created by k8s-ci-robot with trusted label to be automatically merged. Since k8s-ci-robot automatically creates autobump PRs, there is no more manual step needed any more for bumping prow.k8s.io.

Part of: https://github.com/kubernetes/enhancements/tree/master/keps/sig-testing/2539-continuously-deploy-k8s-prow

/hold
Will not merge this until:
- Announced widely in k8s-dev community
- KEP was moved to implementable phase

### CODE REVIEW

**Review by cjwagner — APPROVED** [review]
/hold

**Review by spiffxp — CHANGES_REQUESTED** [review]
Whatever label you decide to use, it should be added to https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml

My preference would be to restrict to the repo it's used for, and (not a strong opinion) make it either:
- yellow (jumps out as visually distinct, humans didn't look at this, not red since it is not blocking merge)
- green (relates to lgtm/approve)
- match whatever color other labels of this prefix use

File: config/prow/config.yaml
Comment (chaodaiG) [reply config/prow/config.yaml:573]: The bot has already signed CLA I believe. And the label is added in a separate PR #21313 , which has been updated to use `skip-review` label instead. Will update this PR once the other one is merged

Comment (spiffxp) [reply config/prow/config.yaml:573]: I might be wrong, but I thought gating merges on the presence of the `cncf-cla: yes` label is how we enforced that all code contributed to this project has come through the CLA.

**Comment by spiffxp** [comment]
Confirmed kubernetes-dev@ has been notified https://groups.google.com/g/kubernetes-dev/c/nke9PYaGTnY/m/nrQ6hVDfCgAJ

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
/hold
remove /hold when ready to deploy

**Comment by chaodaiG** [comment]
/unhold
Let's rock

**Pull request merged**
